### PR TITLE
Adds simple data seeding demo

### PR DESCRIPTION
**Pull request by divega**

### CODE REVIEW

File: data-seeding/data-seeding.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="2.1.0-preview1-27122" />
```
Comment (divega) [inline data-seeding/data-seeding.csproj]: @anpete These are from https://dotnet.myget.org/F/aspnetcore-ci-dev/api/v3/index.json. Any suggestion? Should we leave it floating, add the source to the solution?

File: data-seeding/Migrations/20170925010903_Initial.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using System;
+using System.Collections.Generic;
+
+namespace dataseeding.Migrations
```
Comment (anpete) [inline data-seeding/Migrations/20170925010903_Initial.cs]: Namespace casing

File: data-seeding/Program.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using System.Drawing;
+
+namespace data_seeding
```
Comment (anpete) [inline data-seeding/Program.cs]: Namespace

File: data-seeding/data-seeding.csproj (hunk 1 above)
Comment (anpete) [reply data-seeding/data-seeding.csproj]: Floating. Does this have any effect on the other projects?

**Comment by anpete** [comment]
Needs rebase

**Pull request merged**